src/Symfony/Bundle/SecurityBundle/ValueResolver/SecurityUserValueResolver.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\SecurityBundle\ValueResolver;
```

Comment:
If there is no other files requiring a `HttpKernel` or `Controller` (sub)namespace there is no reason to keep a complex tree in a bundle IMHO.